src/content_mapblock.cpp

```diff
@@ -1316,19 +1315,45 @@ void mapblock_mesh_generate_special(MeshMakeData *data,
 			p2.X++;
 			MapNode n2 = data->m_vmanip.getNodeNoEx(blockpos_nodes + p2);
 			const ContentFeatures *f2 = &nodedef->get(n2);
-			if(f2->drawtype == NDT_FENCELIKE)
+			if(f2->drawtype == NDT_FENCELIKE || f2->drawtype == NDT_NORMAL)
```

Comment:
raillike does it in a broken way. Well, its not broken if you only want to make rails with it, but if you want to have cables, its broken. Technic mv cable doesn't connect with technic LV cable.